vars/sendMetricsMsg.groovy

```diff
@@ -11,7 +11,7 @@ import org.centos.contra.pipeline.Utils
  * @param metricsMap: A map containing the metrics to be sent over the bus
  * @return
  */
-
+string b = ""
```

Comment:
Why is not not inside of the function definition?

Reply:
I was trying to make it global. Since it is only used in call function, I will add it inside the function

Reply:
Yeah I don't see any reason to have it be global